sdk/appconfiguration/app-configuration/test/README.md

```diff
@@ -0,0 +1,10 @@
+# Testing
+
+To test this project, make sure to first build it properly by following our [building instructions](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md#building). Once the project is correctly built, you will be able to run the tests by following the [testing instructions](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md#testing).
+
+The environment variables you will need to properly assign to run the live test of this project are the following:
+
+- `AZ_CONFIG_CONNECTION`: The connection string of your Azure App Configuration account.
```

Comment:
```suggestion
- `TEST_MODE`: live
- `AZ_CONFIG_CONNECTION`: The connection string of your Azure App Configuration account.
```

Reply:
Applies for all other packages that have recorded tests being used by default

Reply:
on it